code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -409,6 +409,7 @@ MATCHBOXES ARE ALSO IN FANCY.DM
 	item_state = "cigar"
 	smoketime = 1500
 	chem_volume = 20
+	species_fit = list(VOX_SHAPED)
```

Comment:
Didn't stop me when I tried it. Joined as human, spawned a cigar, and it loaded the human cigar sprite in my mouth. Lit. Still used human sprite. That's actually the line that makes a lot of things recognize the species/vox folder.